### PR TITLE
Added PotionBrewEvent

### PR DESCRIPTION
**Pull request by thehutch**
I haven't tested this however it looks like it should world. If there is anything I've missed or needs improving please tell me.

Signed-off-by:thehutch

### CODE REVIEW

**Comment by kitskub** [comment]
Formatting...

**Comment by thehutch** [comment]
I know the formatting is different however does it really matter at this time? I'm sure once Vanilla is finished someone will run a formatter over all the code

**Comment by Zidane** [comment]
@thehutch 

Uh yes it does. Its a mess; sorry and no offence but I have no better words. I can't tell what is changed code and what is just a formatter ruining it.

We have a formatting file for IntelliJ (if you use it) or otherwise we can help you adjust your IDE to be compliant with our standards. You can find them here: http://spout.in/prguide

**Comment by thehutch** [comment]
It's not a mess it's code of beauty, i'll change it to Spout's terrible formatting standards. No offence but there's no better way of describing it :D

**Comment by Zidane** [comment]
@thehutch 

Oh? Considering we use oracle standards but instead of spaces we do tabs....that means you are calling their standards bad?

Alright.

**Comment by windy1** [comment]
Make sure you include newlines at the end of files as well.

File: src/main/java/org/spout/vanilla/event/block/PotionBrewEvent.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+	public static HandlerList getHandlerList() {
+		return handlers;
+	}
+
+	@Override
+	public HandlerList getHandlers() {
+		return handlers;
+	}
+}
```
Comment (windy1) [inline src/main/java/org/spout/vanilla/event/block/PotionBrewEvent.java:104]: Needs a new line here.

Comment (thehutch) [inline src/main/java/org/spout/vanilla/event/block/PotionBrewEvent.java:104]: Done

File: src/main/java/org/spout/vanilla/event/block/PotionBrewEvent.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+import org.spout.api.event.Cancellable;
+import org.spout.api.event.Cause;
+import org.spout.api.event.HandlerList;
+import org.spout.api.event.block.BlockEvent;
+import org.spout.api.inventory.ItemStack;
+import org.spout.vanilla.component.block.material.BrewingStand;
+
+/**
+ * @author thehutch
```
Comment (windy1) [inline src/main/java/org/spout/vanilla/event/block/PotionBrewEvent.java]: No author tags please.

File: src/main/java/org/spout/vanilla/event/block/PotionBrewEvent.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
+package org.spout.vanilla.event.block;
```
Comment (windy1) [inline src/main/java/org/spout/vanilla/event/block/PotionBrewEvent.java:27]: Missing header

**Comment by thehutch** [comment]
Git rebasing just isn't working for me, so I can't squash my commits

**Comment by lukespragg** [comment]
@thehutch, regardless of your personal opinions of Oracle coding conventions/standards and ours (tabs vs spaces), you still need to follow them. You also need to squash and actually test your code, else we cannot accept it.

**Pull request closed without merging**